File: Lab2/task32.java
import java.util.Scanner;
public class task32{
	public static void main(String[] args){
		int a, b;
		System.out.println("Please input 2 intergers: ");
		Scanner scan = new Scanner(System.in);
		a = scan.nextInt();
		b = scan.nextInt();
		System.out.printf("%d + %d = %d\n", a, b, a+b);
		System.out.printf("%d - %d = %d\n", a, b, a-b);
		System.out.printf("%d * %d = %d\n", a, b, a*b);
		try{
			System.out.printf("%d / %d = %d\n", a, b, a/b);
		}catch(Exception e){
			System.err.println("The second interger could not be 0.");
		}
	}
}

File: Lab2/task3.2.cpp
#include <stdio.h>
int main(){
	int a, b;
	printf("Please input 2 integers: \n");
	scanf("%d%d", &a, &b);
	if (b == 0){
		fprintf(stderr, "The second interger could not be 0.\n");
		return -1;
	}
	printf("%d + %d = %d\n", a, b, a+b);
	printf("%d - %d = %d\n", a, b, a-b);
	printf("%d * %d = %d\n", a, b, a*b);
	printf("%d / %d = %d\n", a, b, a/b);
	printf("%d %% %d = %d\n", a, b, a%b);
	return 0;
}
